File: scala-gatling-client/src/gatling/scala/io/swagger/client/model/PushWhereObject.scala

package io.swagger.client.model


case class PushWhereObject (
    _deviceType: Option[String],
    _deviceToken: Option[PushWhereObject_deviceToken]
)
object PushWhereObject {
    def toStringBody(var_deviceType: Object, var_deviceToken: Object) =
        s"""
        | {
        | "deviceType":$var_deviceType,"deviceToken":$var_deviceToken
        | }
        """.stripMargin
}

File: dart-jaguar-client/lib/model/push_where_object.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

import 'package:swagger/model/push_where_object_device_token.dart';
part 'push_where_object.jser.dart';

class PushWhereObject {
  
  final String deviceType;
  //enum deviceTypeEnum {  ios,  android,  };
  final PushWhereObjectDeviceToken deviceToken;
  

  PushWhereObject(
    

{
     this.deviceType = null,  
     this.deviceToken = null 
    
    }
  );

  @override
  String toString() {
    return 'PushWhereObject[deviceType=$deviceType, deviceToken=$deviceToken, ]';
  }
}

@GenSerializer()
class PushWhereObjectSerializer extends Serializer<PushWhereObject> with _$PushWhereObjectSerializer {

}
